packages/firebase/index.d.ts

```diff
@@ -7251,14 +7251,32 @@ declare namespace firebase.firestore {
     ): Query;
 
     /**
-     * Creates and returns a new Query where the results are limited to the
-     * specified number of documents.
+     * Creates and returns a new Query where only the first matching documents
+     * are returned as results.
      *
      * @param limit The maximum number of items to return.
      * @return The created Query.
      */
     limit(limit: number): Query;
 
+    /**
+     * Creates and returns a new Query where only the first matching documents
+     * are returned as results.
+     *
+     * @param limit The maximum number of items to return.
+     * @return The created Query.
+     */
+    limitToFirst(limit: number): Query;
```

Comment:
Just FYI- Per the API proposal review, we should keep limit() as-is (don't deprecate it) and not add limitToFirst().  So the only public API change is adding Query.limitToLast().

FWIW I think I would be inclined to still _internally_ differentiate them as limitToFirst / limitToLast though.